.github/ISSUE_TEMPLATE/config.yml

```
@@ -1,7 +1,7 @@
 blank_issues_enabled: true
 contact_links:
-  - name: Conda-forge Matrix/Element.io chat
-    url: https://matrix.to/#/#conda-forge:matrix.org
+  - name: Conda-forge Zulip chat
+    url: https://conda-forge.zulipchat.com
```

Am wondering if we should just link to our outreach page

```suggestion
  - name: Conda-forge community
    url: https://conda-forge.org/community/getting-in-touch/
```